static/css/style.css

```diff
@@ -56,3 +79,9 @@ code {
 .navbar-brand img{
 	height: 100%;
 }
+
+.naslov {
+font-size: 20px; 
```

Comment:
One extra tab in front, and these lines will be aligned just like the other ones in this file :)